tests/lockable_test.py

```diff
@@ -0,0 +1,80 @@
+"""Test locakble decorator."""
+from honeybee._lockable import lockable
+
+import pytest
+
+
+def test_simple_class():
+    """Test the application of the lockable decorator to a simple class."""
+    @lockable
+    class Foo(object):
+        def __init__(self):
+            self.bar = 10
+
+    foo = Foo()
+    foo.bar = 20
+
+    foo.lock()
+    with pytest.raises(AttributeError):
+        foo.bar = 30
+
+    foo.unlock()
+    foo.bar = 30
+    foo.lock()
```

Comment:
you may want to add a quick check here to ensure the value is assigned correctly when unlocked.

Reply:
Good idea. Added